src/index.js

```diff
@@ -8,24 +8,25 @@
 const whitelistUtils = require('./whitelistUtils');
 const expressUtils = require('./expressUtils');
 const port = process.env.PORT || 3000;
+const { logger, initProducer } = require('./logger');
 const listening = `Listening on port ${port}`;
 const API_CACHE_DURATION = process.env.API_CACHE_DURATION || false;
 
 let whitelist;
-
+initProducer();
```

Comment:
It would make it easier to understand what this is if this was named something like "initializeKafkaLoggingProducer" or something like that... just "initProducer" does not give anyone any idea about what this producer is for.

Reply:
fixed